utility/test/utility.cpp

```diff
@@ -176,4 +177,27 @@ SCENARIO("isHexadecimal") {
     }
 }
 
+SCENARIO("binaryCopy") {
+    // integer representations of floats and doubles have been computed using
+    // http://babbage.cs.qc.cuny.edu/IEEE-754/
+
+    float in1 = 1.23456f;
+    // We assume that a float is coded on 32bits. That assumption is made in
+    // the Parameter Framework.
```

Comment:
~~~cpp
THEN("Float should coded on 32bits. That assumption is made in the Parameter Framework.") {
    REQUIRE(sizeof(float) == sizeof(uint32_t));
}
~~~